docs/src/config/core-components.adoc

```diff
@@ -78,235 +83,163 @@ Example:
 unlock_joints_mask=0x38 selects joints 3,4,5
 ----
 
+selecciona articulaciones 3,4,5
```

Comment:
Can this be removed?

Reply:
Yes, please remove in _es and _en. It means "Selects joints 3,4,5". I just copied it since I was uncertain if this should be in or out and the Spanish had it.

Reply:
```suggestion

```